pkg/proxy/iptables/proxier.go

```diff
@@ -1007,6 +992,12 @@ func (proxier *Proxier) syncProxyRules() {
 		proxier.endpointChainsNumber += len(proxier.endpointsMap[svcName])
 	}
 
+	localAddrSet := utilproxy.GetLocalAddrSet()
```

Comment:
When you put these right next to each other, they seem to overlap a bit.  We should clean this up.  Later.

Reply:
which place you think it is overlap? :)